Sources/TuistGenerator/Generator/ConfigGenerator.swift

```diff
@@ -191,8 +191,14 @@ final class ConfigGenerator: ConfigGenerating {
         if let entitlements = target.entitlements {
             settings["CODE_SIGN_ENTITLEMENTS"] = .string("$(SRCROOT)/\(entitlements.relative(to: sourceRootPath).pathString)")
         }
-        settings["SDKROOT"] = .string(target.platform.xcodeSdkRoot)
```

Comment:
This is the meat of the change -- `SDKROOT` and `SUPPORTED_PLATFORMS` are not set which means that values set in XCConfig files can be used.